app/models/concerns/with_financial_incentives.rb

```diff
@@ -0,0 +1,31 @@
+module WithFinancialIncentives
+  extend ActiveSupport::Concern
+
+  included do
+    def has_bursary?
+      dfe_subjects.any?(&:has_bursary?)
+    end
+
+    def has_scholarship_and_bursary?
+      dfe_subjects.any?(&:has_scholarship_and_bursary?)
+    end
+
+    def has_early_career_payments?
+      dfe_subjects.any?(&:has_early_career_payments?)
+    end
+
+    def bursary_amount
+      dfe_subject_which_defines_financial_incentives&.bursary_amount || 0
```

Comment:
there is `NO_FINANCIAL_INCENTIVES`

dfe_subject_which_defines_financial_incentives`&` 

so no need for `&` and `|| 0` ?

Reply:
This is there for when there are no subjects on the course – 32198d988fed33bacdca7ea6f8e15b3da1d3c4a6
This happens a lot in our specs.